### DISCUSSION TITLE
Disaster Recovery

### DISCUSSION THREAD

**Discussion by rubiktubik** [onedr0p/cluster-template#1557] [Q&A] [answered]
Hi awesome community!

How do you do disaster recovery with this template? How can I deploy it to another cluster? 

For volumes I would use velero. But how can can apply my repository to another clean talos cluster?

Regards 
rubiktubik

**Comment by onedr0p** [comment]
I would take a look at [Volsync](https://github.com/backube/volsync), not only does it focus completely on PV / PVC data (unlike Velero), it also has the ability to [restore volumes on bootstrap of your cluster in a hands-off way](https://volsync.readthedocs.io/en/stable/usage/volume-populator/index.html) (also unlike Velero).

I am using it in my [home-ops repo](https://github.com/onedr0p/home-ops) and many others are too.

**Reply by rubiktubik** [reply]
Thank you for your answer Devin!

I've seen that almost everyone in the Homelab community seems to use volsync, but I didn't know it myself.

Thanks for the tip. I will definitely try it out.

What would your disaster recovery scenario look like? How can I restore the cluster with the GitHub repository and Volsync?

**Reply by onedr0p** [reply, marked as answer]
The process with Flux and Volsync is like this:
1. You tell Flux to 🚀 
2. Pods will be in pending because the PVCs are waiting to be populated with the latest volsync (restic) snapshot
3. Eventually volsync will populate the PVCs and the applications come online.
